app/models/snap_application_member_attributes.rb

```diff
@@ -6,7 +6,7 @@ def initialize(member:, position:)
 
   def to_h
     {
-      "#{position}_member_full_name" => member.full_name,
+      "#{position}_member_full_name" => member.display_name,
```

Comment:
note that I kept these hash keys as `full_name` because that is how the fields are named in the PDF and that felt like an accurate name for the info the PDF is looking for. This does bring up the question of whether we should still use `full_name` for the PDF data (the key, in this case) to protect us in the future if we change `display name` to not be the full name